### PR TITLE
Fix consistency of version references in documentation, etc.

### PR DESCRIPTION
**Pull request by johngmyers**
### Issue

When the 2.4.5 release was cut, not all references to the 2.4.4 version were updated.

### Description

Creates a `hack/set-version` script for updating the version number of a release. Adds a check to GitHub actions to ensure all versions in the repo are set consistently.

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `docs` directory)
- [x] Manually tested
- [x] Made sure the title of the PR is a good description that can go into the release notes

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: docs/deploy/installation.md [hunk 1]
```diff
@@ -190,7 +190,7 @@ We recommend using the Helm chart. This supports Fargate and facilitates updatin
     ### Apply YAML
     1. Download spec for load balancer controller.
     ```
-    wget https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.4/v2_4_4_full.yaml
+    wget https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.5/v2_4_4_full.yaml
```
Comment (kishorj) [inline docs/deploy/installation.md:193]: v2_4_5_full.yaml

File: docs/deploy/installation.md [hunk 2]
```diff
@@ -218,7 +218,7 @@ We recommend using the Helm chart. This supports Fargate and facilitates updatin
     ```
     1. Optionally download the default ingressclass and ingressclass params
     ```
-    wget https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.4/v2_4_4_ingclass.yaml
+    wget https://github.com/kubernetes-sigs/aws-load-balancer-controller/releases/download/v2.4.5/v2_4_4_ingclass.yaml
```
Comment (kishorj) [inline docs/deploy/installation.md:221]: v_2_4_5_ingclass.yaml

**Review by kishorj — COMMENTED** [review]
Thanks a bunch. I have some minor comments regarding themanifest name

File: docs/deploy/installation.md (hunk 1 above)
Comment (johngmyers) [reply docs/deploy/installation.md]: Fixed

**Comment by kishorj** [comment]
I need to investigate the e2e failures, please don't flag for retest.

**Pull request merged**